### PR TITLE
firefox-esr linuxbinary formula

### PR DESCRIPTION
**Pull request by emanresusername**
https://github.com/athrunsun/homebrew-linuxbinary/pull/19

### CODE REVIEW

File: homebrew-linuxbinary/Formula/firefox-esr.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class FirefoxEsr < Formula
+    desc 'The 100% fresh, free-range, ethical browser, with more speed, privacy and freedom. This is the ESR version, in en-US.'
+    homepage "https://www.mozilla.org/en-US/firefox/beta/all/"
```
Comment (athrunsun) [inline homebrew-linuxbinary/Formula/firefox-esr.rb]: @emanresusername I think the homepage should be `https://www.mozilla.org/en-US/firefox/organizations/all/`.

**Comment by athrunsun** [comment]
@emanresusername Sorry I was disrupted during the weekend and didn't open PRs on your behalf on time. Requested a change for the homepage, could you take a look?

**Comment by athrunsun** [comment]
@emanresusername Another thing is, please ignore the "All checks have failed" message - I shouldn't build against PR in travis, it should only be used to push changes in master branch to respective repo (this is fixed now).

File: homebrew-linuxbinary/Formula/firefox-esr.rb (hunk 1 above)
Comment (emanresusername) [reply homebrew-linuxbinary/Formula/firefox-esr.rb]: indeed it should 👍

**Pull request merged**